### PR TITLE
Cookie policy, Tag manager & Crumb CSRF

### PR DESCRIPTION
**Pull request by govTechSatish**
Added Cookie policy, Tag manager & Crumb CSRF

### CODE REVIEW

File: test/integration/narrow/routes/collaboration.test.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
+const { getCookieHeader, getCrumbCookie } = require('./test-helper')
 describe('Collaboration page', () => {
+  process.env.COOKIE_PASSWORD = '1234567890123456789012345678901234567890'
```
Comment (paulsimonandrews) [inline test/integration/narrow/routes/collaboration.test.js:3]: This is not needed as it's set in `docker-compose.yaml`. Same for all the other tests below.

File: app/config/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+const Joi = require('joi')
+const mqConfig = require('./messaging')
+
+// Define config schema
+const schema = Joi.object({
+  port: Joi.number().default(3000),
```
Comment (paulsimonandrews) [inline app/config/index.js:6]: Most of this config isn't used and is therefore redundant. I'd prefer not to add config that we aren't using, and to only add it when the features that use it are added.

From what I can tell we only need `cookiePassword`, `cookieOptions` and `googleTagManagerKey` in this PR, so I think the rest should be removed.

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Looking good.

A few changes requested, mostly about removing code we don't need.

File: app/templates/cookies/cookie-policy.njk [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
```
[the remark this answers is not in the document]
Comment (govTechSatish) [reply app/templates/cookies/cookie-policy.njk]: resolved.

**Review by paulsimonandrews — APPROVED** [review]
LGTM 👍

**Pull request merged**